### ISSUE TITLE
Improve support for full-day-events

### ISSUE THREAD

**Issue by CooperBarrett** [tobixen/calendar-cli#49]
Hello, I have a quick question, 
Is it possible to add a "full day" event? I am not talking about a 24-hour event, I do not know if I am clear...
**Event Full Day** : 
![](https://framapic.org/xmO8UndH3ZqG/AaaEvK4RoQLR)
**Event 24h (23h59)** : 
![](https://framapic.org/ExxnCxM5QDu6/efajtWd1rfrq)

**Comment by tobixen** [comment]
That's an easy question, just use the parameter --whole-day (--full-day is probably more correct English?)

    calendar-cli calendar add --whole-day '2019-12-24+1d' 'julaften'

Without the "1d" it breaks because the default duration is set to 1h.

Apparently I did not think at all about whole-day-events when I wrote calendar-cli.  According to the standard, a full-day-event is just an ordinary event specified with dates rather than timestamps.  From this perspective, feed calendar-cli with dates rather than timestamps and it ought to work:

    calendar-cli calendar add --whole-day '2019-12-24' 'julaften'

Except, apparently I have done quite some hard-coding on timestamps, so that's fragile.

This ought to be cleaned out a bit, leave the bug open.
